.github/workflows/release-compose.yml

```
@@ -0,0 +1,80 @@
+      - name: Package Custom Artifact
+        id: package_artifact
+        run: |
+          docker_artifact="csghub-docker-compose-${{ github.ref_name }}.tgz"
+          chart_artifact="csghub-helm-chart-${{ github.ref_name }}.tgz"
+          mv docker-compose csghub && tar -zcf $docker_artifact csghub
+          tar -zcf $chart_artifact helm-chart/charts/csghub
+          echo "artifacts=$docker_artifact,$chart_artifact" >> $GITHUB_OUTPUT
+
+      - name: Check if Release Exists
+        id: check_if_exists
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:33: Double quote to prevent globbing and word splitting

(shellcheck)